src/edu/csus/ecs/pc2/util/OSCompatibilityUtilities.java

```diff
@@ -134,7 +134,7 @@ public static boolean areSandboxesSupported(IInternalContest contest, Log log)
         String home = versionInfo.locateHome();
         
         //point to the PC2 Internal Sandbox file (under "/sandbox" in the home, i.e. installation, directory)
```

Comment:
The above comment should be changed to reference the `scripts` folder instead of the `sandbox` folder.

Reply:
Changed in next commit.